daceml/pytorch/module.py

```diff
@@ -0,0 +1,64 @@
+try:
+    import torch
+except ModuleNotFoundError:
+    raise ImportError("To use the torch frontend, please install torch")
+
+import os
+import tempfile
+from functools import wraps
+
+import torch.nn as nn
+import onnx
+
+from daceml.onnx import ONNXModel
+from daceml.onnx.shape_inference import infer_shapes
+
+
+class DACEModule(nn.Module):
```

Comment:
Naming convention is a bit weird. Please use either `DaceModule` (standard Python) or `DaCeModule`.